DQM/EcalMonitorTasks/python/EcalMonitorTask_cff.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQM.EcalMonitorTasks.EcalMonitorTask_cfi import *
+
+# Customization to run the CPU vs GPU comparison task if the job runs on a GPU enabled machine
+from Configuration.ProcessModifiers.gpuValidationEcal_cff import gpuValidationEcal
+from DQM.EcalMonitorTasks.GpuTask_cfi import ecalGpuTask
```

Comment:
Can you rename `DQM/EcalMonitorTasks/python/GpuTask_cfi.py` to `DQM/EcalMonitorTasks/python/ecalGpuTask_cfi.py`, so the file name matches the main object defined in it ?